bin/check_samplesheet.py

```diff
@@ -102,15 +94,15 @@ def check_samplesheet(file_in, file_out):
                         )
 
             ## Auto-detect paired-end/single-end
-            sample_info = []  ## [single_end, fastq_1, fastq_2]
+            sample_info = []  ## [single_end, lane, fastq_1, fastq_2, sex, phenotype, paternal_id, maternal_id, case_id]
```

Comment:
Maxime told me that sarek is using 'gender' instead of 'sex' as keyword. Let's try to use the same nomenclature when we can/remember 😄

Reply:
@pbiology